roombapy/const.py

```diff
@@ -1,6 +1,12 @@
 """Constants for roombapy."""
+from __future__ import annotations
 
-MQTT_ERROR_MESSAGES = {
+TransportErrorCode = int
+TransportErrorMessage = str | None
+ErrorCode = int
+ErrorMessage = str
```

Comment:
Would it make sense to make these an Enum? `IntEnum` or `StrEnum`?

Reply:
In future — yes, sure.